DEVGUIDE.md

```diff
@@ -34,22 +34,27 @@ STARGATE_PASSWORD=cassandra
 ```shell
 npm run test
 ```
+## Update Stargate and JSON API versions
+
+Stargate and the JSON API versions are maintained in the file `api-compatibility.versions`. Update the versions accordingly, submit a PR and make sure that the Github Actions that verify the new versions run fine.
```

Comment:
Couple of thoughts:

1) Would be great to put API compatibility in the docs
2) Maybe stargate-mongoose can throw an error if connecting to an unsupported version of JSON API or Stargate?

Reply:
1. I referred the versions file (i.e. api-compatibility.versions) in the README as well (https://github.com/stargate/stargate-mongoose#version-compatibility). The idea is that if we have to update the version it will be only one place instead of updating in multiple places. This file is used by the CI workflows as well as referred in the README and DEVGUIDE files.
2. Regarding throwing an error, I think we need a versions API from the JSON API as well, so on establishing the connection, we can check the server version and compare it with what this driver has been tested with (by looking up api-compatibility.versions probably) and throw an error if required.